docs/source/tutorials/visualising_scaffolds/visualising_scaffolds.rst

```diff
@@ -1,21 +1,50 @@
-*******
-Example
-*******
+Example of visualising scaffolds
+================================
 
-.. toctree::
-   :maxdepth: 2
+In this example, we will visualize a scaffold, modify it using user interface, and then export its nodal parameters.
 
-=======
-Case 1 :
-=======
+Once you have successfully installed and executed a new workflow, you should see ``scaffoldmaker`` window open.
+In the left top corner, find :guilabel:`Mesh type` drop down list, and choose :guilabel:`3D Bladder with Urethra 1`.
+Often, there are more than one version of parameter set
+for each mesh type. Let's click on :guilabel:`Parameter set`, and select :guilabel:`Rat 1`.
 
-Model
------
+Move your mouse upward while holding :kbd:`right mouse button` to zoom in, and
+rotate the object by moving your mouse while holding :kbd:`left mouse button`.
 
+.. image:: ../../images/visualization_scaffold/vs2.png
+    :class: with-shadow
 
-=======
-Case 2 :
-=======
+You may notice there is a :guilabel:`Central path LUT` option under :guilabel:`Parameter set`
+. Some mesh types have a special option which allows you
+to modify multiple related nodes using a simpler geometry structure.
+Click on it, you should see a 1D line mesh representing the urethra. Find :guilabel:`Node points` in the bottom left corner,
+and make sure the box is ticked. Hold :kbd:`S` key, and you can drag node points using your :kbd:`left mouse button`.
+
+.. image:: ../../images/visualization_scaffold/vs4.png
+    :class: with-shadow
+
+Once you are happy with the shape, click :guilabel:`<< Back` on the top left corner. You should see the shape
+of urethra has already changed.
+
+.. image:: ../../images/visualization_scaffold/vs3.png
+    :class: with-shadow
+
+You may found some meshes are quite big. If you see part of the mesh exceeds the coordinates system's limitation and disappears on screen,
```

Comment:
"If part or all of the mesh is not visible in the field of view of the scaffoldmaker's graphics viewport, you can rescale the mesh by holding ....